docs/vertex_ai/VertexAIReadme.md

```diff
@@ -0,0 +1,20 @@
+Get Started with Vertex AI in Firebase
+===========================================
+
+Thank you for installing the Vertex AI in Firebase Unity SDK.
+
+The Vertex AI Gemini API gives you access to the latest generative AI models from Google: the Gemini models. This SDK is built specifically for use with Unity and mobile developers, offering security options against unauthorized clients as well as integrations with other Firebase services.
+
+With this, you can add AI personalization to your app, build an AI chat experience, create AI-powered optimizations and automation, and much more!
+
+### Links
+
+* [Homepage](https://firebase.google.com/games/)
```

Comment:
should we use games page or vertex page? maybe after we have unity code snippet ready? https://firebase.google.com/docs/vertex-ai

Reply:
Good idea, added it.